compiler/src/iree/compiler/Dialect/Flow/Transforms/test/normalize-input-indexing-map.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: iree-opt --split-input-file --verify-diagnostics --iree-flow-interchange-transpose-generic-ops --canonicalize -cse %s | FileCheck %s
```

Comment:
normalize-input-indexing-map.mlir -> interchange_transpose_generic_ops.mlir

I think the convention is that the test file name should match the pass name. And we should match the existing style -- use `_` instead.